sycl/doc/extensions/usm/usm.adoc

```diff
@@ -325,21 +372,17 @@ Parameters::
   * `size_t count` - number of bytes requested to be prefetched
 Return value:: none
 
-=== Concurrent USM
-Concurrent USM builds upon Restricted USM by enabling concurrent access to `shared` allocations between host and devices.  Additionally, some implementations may support a working set of `shared` allocations larger than device memory.
-
-==== Concurrent USM Utility Functions
-Concurrent USM contains all the utility functions of Explicit USM and Restricted USM.  It introduces a new function, `sycl_mem_advise`, that allows programmers to provide additional information to the underlying runtime about how different allocations are used.  
-
-==== Performance Hints
+'''
+==== Concurrent USM
+Concurrent USM contains all the utility functions of Explicit USM and Restricted USM.  It introduces a new function, `sycl::mem_advise`, that allows programmers to provide additional information to the underlying runtime about how different allocations are used.  
 
-==== prefetch
+===== Performance Hints
+===== prefetch
 In Concurrent USM, prefetch commands may be overlapped with kernel execution.
-
-==== sycl_mem_advise
+===== mem_advise
```

Comment:
Maybe it's just a GitHub artifact, but I see following in the rendered version:

In Concurrent USM, prefetch commands may be overlapped with kernel execution. ===== mem_advise

Reply:
Good catch.